lib/convert.js

```diff
@@ -14,8 +15,32 @@ var converter = {
     currentFolder: {},
     env: {},
 
-    getMetaData: function(ramlString, callback) {
-        raml.load(ramlString).then(function(data) {
+    getMetaData: function(input, callback) {
+        var allFiles = input.allFiles,
+            reader = new ramlParser.FileReader(function (filePath) {
+                return new Promise(function (resolve, reject) {
+                    var targetFilePath = decodeURIComponent(filePath);
+
+                    if (process.platform == 'win32' || process.platform == 'win64') {

```

Comment:
@VShingala This will make this function unusable on the browser. Should have a check for presence of the process property.